fluentd/out_syslog.rb

```diff
@@ -99,7 +99,7 @@ def emit(tag, es, chain)
         end
         @packet.time = time
         @packet.tag = @random_string
-        if @tag_key.any? && record.any?
+        if !@tag_key.empty? && record.any?
```

Comment:
What are you trying to determine about `record` here?  Are you trying to test to see if record has any fields in it?  If so, you should use `!record.empty?` instead.

Reply:
Thanks, @richm !  Pushed a new patch...